Sources/DbPackages-mysql.php

```diff
@@ -226,6 +226,8 @@ function smf_db_create_table($table_name, $columns, $indexes = array(), $paramet
 	$table_query .= ') ENGINE=' . $parameters['engine'];
 	if (!empty($db_character_set) && $db_character_set == 'utf8')
 		$table_query .= ' DEFAULT CHARSET=utf8 COLLATE=utf8_general_ci';
+	else
```

Comment:
Maybe an esle if would be better here?

Reply:
It's either 'utf8' (currently an alias of 'utf8mb3') or 'utf8mb4'.  These are the only two options at this point.  And as noted in the description to this PR, we are defaulting to 'utf8mb4' going forward.